Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
The software is stable for >7 years now. It does lack proper release engineering practices but that does not mean it is useless.

It would be nice if you can document this new requirement of "no build system" == "no homebrew" for the benefits of others that might fall in the same trap as I did.

Feel free to close this pull request as I won't be pursuing it further given this new set of requirements.